CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ If you are updating tests and just want to run a single test to check it, you
 can use this syntax to run it exactly as the test harness would:
 
 ```text
-$ python tools/test.py -v --mode=release parallel/test-stream2-transform
+$ python tools/test.py --mode=release parallel/test-stream2-transform
```

Comment:
> The reason I think this is necessary is that using -J can cause subtle bugs (when tests interact with each other)

Yeah, I agree with that if we are talking about running multiple tests. But that sentence is about running a single test, isn't it? Do I miss the point?